frontend/package.json

```diff
@@ -41,19 +41,19 @@
     "storybook": "storybook dev -p 6006 -s ../config"
   },
   "dependencies": {
-    "@openshift/dynamic-plugin-sdk": "^3.0.0",
-    "@openshift/dynamic-plugin-sdk-utils": "^1.2.0",
-    "@patternfly/patternfly": "4.224.2",
-    "@patternfly/quickstarts": "^2.3.1",
-    "@patternfly/react-catalog-view-extension": "4.95.1",
-    "@patternfly/react-charts": "6.94.18",
-    "@patternfly/react-code-editor": "^4.82.115",
-    "@patternfly/react-core": "4.276.6",
-    "@patternfly/react-icons": "4.93.6",
-    "@patternfly/react-styles": "4.92.6",
-    "@patternfly/react-table": "4.112.39",
-    "@patternfly/react-topology": "^4.91.27",
-    "@patternfly/react-virtualized-extension": "4.88.113",
+    "@openshift/dynamic-plugin-sdk": "^4.0.0",
+    "@openshift/dynamic-plugin-sdk-utils": "^4.0.0",
```

Comment:
Upgrading these requires some code changes too, as `PluginLoader` is removed in the [latest release](https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/lib-core/CHANGELOG.md#400---2023-04-13) 

https://github.com/jenny-s51/odh-dashboard/blob/a079eeaf9dd45d7ba37814c9321d3770c94d77ca/frontend/src/SDKInitialize.tsx#L3

cc @andrewballantyne

Reply:
@jenny-s51 let me know if you need help addressing dependencies (like this). The team will be happy to help out as needed.

Reply:
Thank you @andrewballantyne , it would be awesome if the team could lean in and help make the necessary code changes related to the removal of `PluginLoader`

Reply:
@jenny-s51 see here for a similar change:
https://github.com/opendatahub-io/odh-dashboard/pull/1750/files#diff-588645cf00d4be2a5c5231ce921aa780a581cebdd9f4ef97efd51f6330bc2787

![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/aab85fca-50ae-4511-9781-23e8fec75c87)